src/forms/workflow-form.schema.js

```diff
@@ -0,0 +1,64 @@
+import { componentTypes, validatorTypes } from '@data-driven-forms/react-form-renderer';
+
+export const createWorkflowSchema = (newRecord, rbacGroups) => ({
+  fields: [{
+    component: componentTypes.WIZARD,
+    name: 'workflow_wizard',
+    assignFieldProvider: true,
```

Comment:
This is no longer necessary FieldProvider is passed by default to every component.

Reply:
ok, removed